resources/deb/triggers.erb

```diff
@@ -0,0 +1,17 @@
+<%- unless get_activate_triggers.empty? -%>
```

Comment:
I don't think we need to test if this is empty since `for-each`'ing over an empty array should have the same effect.

Reply:
oh.. if I had scrolled down I would have seen Ryan's comment.